kasa/tests/test_childdevice.py

```diff
@@ -60,6 +60,11 @@ def _test_property_getters():
         )
         for prop in properties:
             name, _ = prop
+            # Skip emeter and time properties
+            # TODO: needs API cleanup, emeter* should probably be removed in favor
+            #  of access through features/modules, handling of time* needs decision.
```

Comment:
At the moment, the child device inherits all properties from the smartdevice class, but may not necessarily have the same modules available. For emeter* the problem solves by itself when/if we remove those properties, but unsure how to proceed with other properties like `time` and `timezone`. Fallback to parent's information?